qa/suites/rbd/basic/cachepool/small.yaml

```diff
@@ -1,5 +1,5 @@
 overrides:
-  ceph:
+  ceph: 
```

Comment:
i think regex might have introduced these extra whitespaces, can you fix them

Reply:
Sure I will check this and rectify